DependencyInjection/Configuration.php

```diff
@@ -110,6 +110,7 @@ public function getConfigTreeBuilder(): TreeBuilder
                             ->prototype('scalar')->end()
                         ->end()
                         ->booleanNode('serialize_null')->defaultFalse()->end()
+                        ->booleanNode('disable_custom_jms_registry')->defaultFalse()->end()
```

Comment:
To be in line with the names of the other config options I think I would rename this one to `enable_jms_registry` and set it to `true` by default.

Reply:
I agree, it's updated. Thanks!